.github/workflows/pitest-update-pr.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-22.04
     steps:
+      - name: Install Harden-Runner
+        uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
+        with:
+          # XXX: Replace with `block` policy.
+          egress-policy: audit
```

Comment:
Here and below: to be updated once we triggered these jobs from `master`. Easiest way to test.